shell/common/platform_view_service_protocol.cc

```diff
@@ -314,4 +316,29 @@ void PlatformViewServiceProtocol::ScreenshotGpuTask(SkBitmap* bitmap) {
   canvas->flush();
 }
 
+const char* PlatformViewServiceProtocol::kWaitUIThreadIdleExtensionName =
+    "_flutter.waitUIThreadIdle";
```

Comment:
Are you sure? The implementation calls `blink::Threads::UI()->PostTask`, which will queue up a task. Given that the message queue is FIFO, all tasks that were scheduled prior to this task will execute first. No?